include/librealsense2/h/rs_device.h

```diff
@@ -335,16 +335,17 @@ const rs2_raw_data_buffer* rs2_run_on_chip_calibration(rs2_device* device, const
                                     }
                                     average step count - number of frames to average, must be between 1 - 30, default = 20
                                     step count - max iteration steps, must be between 5 - 30, default = 10
-                                    accuracy - Subpixel accuracy level, value can be one of: Very high = 0 (0.025%), High = 1 (0.05%), Medium = 2 (0.1%), Low = 3 (0.2%), Default = Very high (0.025%), default is very high (0.025%)
+                                    accuracy - Subpixel accuracy level, value can be one of: Very high = 0 (0.025%), High = 1 (0.05%), Medium = 2 (0.1%), Low = 3 (0.2%), Default = Very high (0.025%), default is Medium
                                     scan_parameter - value can be one of: Py scan (default) = 0, Rx scan = 1
                                     data_sampling - value can be one of:polling data sampling = 0, interrupt data sampling = 1
                                     if json is nullptr it will be ignored and calibration will use the default parameters
-* \param[in]  content_size        Json string size if its 0 the json will be ignored and calibration will use the default parameters
-* \param[in]  callback            Optional callback to get progress notifications
-* \param[in] timeout_ms          Timeout in ms (use 5000 msec unless instructed otherwise)
-* \return                         New calibration table
+* \param[in]  content_size       Json string size if its 0 the json will be ignored and calibration will use the default parameters
+* \param[in]  callback           Optional callback to get progress notifications
+* \param[in]  timeout_ms         Timeout in ms (use 5000 msec unless instructed otherwise)
+* \param[out] health             The health check numbers before and after calibration
+* \return                        New calibration table
 */
-const rs2_raw_data_buffer* rs2_run_tare_calibration_cpp(rs2_device* dev, float ground_truth_mm, const void* json_content, int content_size, rs2_update_progress_callback* progress_callback, int timeout_ms, rs2_error** error);
+const rs2_raw_data_buffer* rs2_run_tare_calibration_cpp(rs2_device* dev, float ground_truth_mm, const void* json_content, int content_size, float* health, rs2_update_progress_callback* progress_callback, int timeout_ms, rs2_error** error);
```

Comment:
API change is reverted since the tare health check numbers feature is not in official version of firmware yet. A new pull request will be created when firmware is ready.